app/Console/Kernel.php

```diff
@@ -26,9 +26,9 @@ class Kernel extends ConsoleKernel
      */
     protected function schedule(Schedule $schedule)
     {
-        $schedule->job(new PickupBookJob())->dailyAt('12:00');
+        $schedule->job(new PickupBookJob())->hourly();
```

Comment:
If we send this every hour, it will send 23 emails for PickupBookMail

Reply:
@Zelig880
The reason I made it hourly was simply cos of the request being open for you 24 hours... So it has to be run every hour to capture that but I think I'll just create a separate cron job for that then.